### DISCUSSION TITLE
Need help: timestamp

### DISCUSSION THREAD

**Discussion by LinTechSo** [vectordotdev/vector#17568] [Q&A]
Hi.

Assumed that i have three vector pipeline in different locations in k8s clusters and my logs transferred between them

vector01 --> vector02 --> vector03 --> Loki

Finally my logs inserted in Loki data store and i received my logs in Loki with  the timestamp from vector (remove_timestamp: false)

My question is this timestamp comes from which vector ? i mean when did this timestamp created on which vector ? is it overwrite  with the newest time at each vector?

**Comment by jszwedko** [comment]
You could use a `remap` transform in vector03 to set the timestamp to the current timestamp. I believe you will otherwise get the timestamp from vector01.
